### PR TITLE
Symfony 3 3 upgrade

### PR DESCRIPTION
**Pull request by akhromets**
Upgrade to compatibility with Symfony 3.3

### CODE REVIEW

File: Resources/config/services.yml [hunk 1]
```diff
@@ -1,9 +1,8 @@
 
 services:
-  happyr.linkedin:
-    class: Happyr\LinkedIn\LinkedIn
+  Happyr\LinkedIn\LinkedIn:
```
Comment (Nyholm) [inline Resources/config/services.yml:3]: Why do we need to update the service id?

**Review by Nyholm — COMMENTED** [review]
Great. Thank you.

File: Resources/config/services.yml (hunk 1 above)
Comment (akhromets) [reply Resources/config/services.yml:3]: http://take.ms/PtQTq
https://symfony.com/doc/current/service_container/3.3-di-changes.html

Comment (akhromets) [reply Resources/config/services.yml:3]: And also a few log messages about deprecating:
http://take.ms/qYJLx
http://take.ms/kYEyP

Comment (Nyholm) [reply Resources/config/services.yml:3]: Yeah, Im very aware of those changes. Could you please just add the quotes around the parameters. 

And then to fix the autowire thing we need to do something like: 

```
services: 
  Happyr\LinkedIn\LinkedIn:
    alias: happyr.linkedin
    public: false
```

Comment (akhromets) [reply Resources/config/services.yml:3]: Of course, there must be backward compatibility! Thank you for remind.

**Pull request closed without merging**